config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/mechanics/:id", to: "mechanics#show"
+  post "/mechanics/:id", to: "ride_mechanics#create"
```

Comment:
Great controller organization. This specific path could be made slightly more RESTful like so: "/mechanics/:mechanic_id/rides" in order to indicate that you are creating a joins record between mechanics and rides (vs. a path that only includes the mechanics resource in it)